tests/test_namespace.py

```diff
@@ -0,0 +1,114 @@
+import screenpy
+
+
+def test_namespace():
+    expected = [
+        'Actor',
+        'AnActor',
+        'Assert',
+        'AssertAllOf',
+        'AssertAnyOf',
+        'AttachAFile',
+        'AttachFile',
+        'AttachTheFile',
+        'Confirm',
+        'ConfirmAllOf',
+        'ConfirmAnyOf',
+        'ContainItemMatching',
+        'ContainTheEntries',
+        'ContainTheEntry',
+        'ContainTheItem',
+        'ContainTheKey',
+        'ContainTheText',
+        'ContainTheValue',
+        'ContainsItemMatching',
+        'ContainsTheEntries',
+        'ContainsTheEntry',
+        'ContainsTheItem',
+        'ContainsTheKey',
+        'ContainsTheText',
+        'ContainsTheValue',
+        'Debug',
+        'Director',
+        'DoNot',
+        'DoesNot',
+        'Empty',
+        'EndWith',
+        'EndsWith',
+        'Equal',
+        'EqualTo',
+        'Equals',
+        'Eventually',
+        'GreaterThan',
+        'GreaterThanOrEqualTo',
+        'HasLength',
+        'InRange',
+        'IsCloseTo',
+        'IsEmpty',
+        'IsEqual',
+        'IsEqualTo',
+        'IsGreaterThan',
+        'IsGreaterThanOrEqualTo',
+        'IsInRange',
+        'IsLessThan',
+        'IsLessThanOrEqualTo',
+        'IsNot',
+        'LessThan',
+        'LessThanOrEqualTo',
+        'MakeNote',
+        'Match',
+        'Matches',
+        'Narrator',
+        'Observe',
+        'ObserveAllOf',
+        'ObserveAnyOf',
+        'Pause',
+        'ReadExactly',
+        'ReadsExactly',
+        'See',
+        'SeeAllOf',
+        'SeeAnyOf',
+        'Sleep',
+        'StartWith',
+        'StartsWith',
+        'TakeNote',
+        'Verify',
+        'VerifyAllOf',
+        'VerifyAnyOf',
+        '__all__',
+        '__builtins__',
+        '__cached__',
+        '__doc__',
+        '__file__',
+        '__loader__',
+        '__name__',
+        '__package__',
+        '__path__',
+        '__spec__',
+        'act',
+        'actions',
+        'actor',
+        'and_',
+        'aside',
+        'beat',
+        'directions',
+        'director',
+        'exceptions',
+        'given',
+        'given_that',
+        'given_when_then',
+        'narration',
+        'noted',
+        'noted_under',
+        'pacing',
+        'protocols',
+        'resolutions',
+        'scene',
+        'settings',
+        'speech_tools',
+        'the_narrator',
+        'the_noted',
+        'then',
+        'when',
+    ]
+    assert dir(screenpy).sort() == expected.sort()
```

Comment:
Hm... i think i know the intended purpose of this test, to make sure we update the `__init__.py` file when we have new stuff to add. However, i'm not sure this will do that. I think, for example, even if we add more Actions and add them to the `__all__` in `screenpy/actions/`, this test will pass.

All that to say, i'm not sure this test is that valuable, but i could be missing an angle.

Reply:
This will fail if there are any changes to the module unless the `expected` list is also updated since it does a `__eq__` between the two lists.

Reply:
OH WHOOPS!!!  that's a really bad test.  `dir(screenpy).sort() == expected.sort()` --> `None == None`